client/rhel/mgr-daemon/mgr-daemon.spec

```diff
@@ -28,21 +35,25 @@ Version:        4.0.1
 Release:        1%{?dist}
 Summary:        Spacewalk query daemon
 License:        GPL-2.0-only
+%if %{_vendor} == "debbuild"
+Group:          utils
+Packager:       Spacewalk Project <spacewalk-devel@redhat.com>
+%else
 Group:          System Environment/Base
+%endif
 Source0:        spacewalksd-%{version}.tar.gz
 Source1:        %{name}-rpmlintrc
 URL:            https://fedorahosted.org/spacewalk
```

Comment:
Spacewalk URL is wrong, and should be fixed across the board.

Reply:
done